installscripts/cookbooks/jenkins/files/node/jazz-installer-vars.json

```diff
@@ -93,5 +93,17 @@
 			 "logs": false,
 			 "assets": false
 		 }
-   }
+   },
+    "APIGEE": {
+        "endpoints": {
```

Comment:
Let's have these configurations structured similar to how we have for AWS API Gateway. 
"APIGEE": {
  	"API_ENDPOINTS": {
    		"PROD": {APIGEE_PROD_API_ENDPOINT} ......